src/components/editor/Bindings/Editor.tsx

```diff
@@ -10,16 +24,129 @@ interface Props {
-                collectionName={currentCollection}
-                readOnly={readOnly}
-            />
+            <Box sx={{ p: 1 }}>
+                <BindingsTabs
+                    selectedTab={activeTab}
+                    setSelectedTab={setActiveTab}
+                />
+
+                <Box sx={{ p: 1 }}>
+                    {tabProps[activeTab].value === 'config' ? (
+                        <ResourceConfig
+                            collectionName={currentCollection}
+                            readOnly={readOnly}
+                        />
+                    ) : collectionData || collectionData === null ? (
+                        <Stack
+                            spacing={2}
+                            sx={{
+                                '& .react-json-view': {
+                                    backgroundColor: 'transparent !important',
+                                },
+                            }}
+                        >
+                            {schemaUpdateErrored ? (
+                                <AlertBox severity="warning" short>
+                                    <FormattedMessage id="workflows.collectionSelector.alert.message.schemaUpdateError" />
+                                </AlertBox>
+                            ) : null}
+
+                            {persistedDraftId ? (
+                                <Box
+                                    sx={{
+                                        display: 'flex',
+                                        alignItems: 'center',
+                                        justifyContent: 'space-between',
+                                    }}
+                                >
+                                    <Box
+                                        sx={{
+                                            display: 'inline-flex',
+                                            alignItems: 'center',
+                                        }}
+                                    >
+                                        <Typography variant="h6" sx={{ mr: 1 }}>
+                                            <FormattedMessage id="workflows.collectionSelector.header.collectionSchema" />
+                                        </Typography>
+
+                                        {/* {schemaUpdated ? (
+                                            <IconButton
+                                                onClick={handlers.updateSchema}
+                                            >
+                                                <Refresh />
+                                            </IconButton>
+                                        ) : (
+                                            <CircularProgress
+                                                size="1.5rem"
+                                                sx={{ ml: 1 }}
+                                            />
+                                        )} */}
```

Comment:
Can we add a TODO to document why this is commented out.

Reply:
This comment was intended to be temporary as I searched for an alternative location for the loading indicator. The header of the monaco editor proved to be an apt placement (in the interim).